universal-application-tool-0.0.1/test/support/ProgramBuilderTest.java

```diff
@@ -70,6 +71,7 @@ public void createProgramWithQuestions() throws Exception {
             .setQuestionType(QuestionType.NAME)
             .setQuestionText(ImmutableMap.of(Locale.US, "question?"))
             .setQuestionHelpText(ImmutableMap.of(Locale.US, "help text"))
+            .setLifecycleStage(LifecycleStage.DRAFT)
```

Comment:
Question (blocking for an answer): should this be ACTIVE? what is the difference/implication of using DRAFT here?

Reply:
DRAFT is correct here, because the later tests do not make the question active by publishing a program containing it.